modules.json

```diff
@@ -332,5 +332,15 @@
             "format": "json",
             "modules": "parse|query+info|query+blah"
         }
+    },
+    {
+        "filename": "feedwatchlist_other.py",
```

Comment:
File name is different..

Reply:
Fixed in https://github.com/srish/MediaWiki-Action-API-Code-Samples/pull/42/commits/6d0834180dc5c959cb016f363a1c0eb046357ba5